test/src/Lib/MockManticoreServer.php

```diff
@@ -14,7 +14,9 @@
 use Manticoresearch\Buddy\Enum\ManticoreEndpoint;
 use Manticoresearch\Buddy\Exception\SocketError;
 use RuntimeException;
+//@codingStandardsIgnoreStart
```

Comment:
We should remove comments like this due to it's fixed already in the ruleset.xml, and PHPCS will not remove them when they are used only in comment blocks for type hinting